eZ/Publish/Core/Persistence/Cache/UserHandler.php

```diff
@@ -58,10 +58,10 @@ public function init(): void
         $this->getRoleTags = static function (Role $role) {
             return ['role-' . $role->id];
         };
-        $this->getRoleKeys = static function (Role $role) {
+        $this->getRoleKeys = function (Role $role) {
```

Comment:
Why removing `static`?

Reply:
to use the escapeForCacheKey function we need to change from a static function to normal